app/main.py

```diff
@@ -2,0 +1,42 @@
+from types import TracebackType
+
+
+class Animal:
+    alive = []
+
+    def __init__(self, name: str, health: int = 100) -> None:
+        self.name: str = name
```

Comment:
Code Style Issue: Good job on using annotation. This is a good practice as it helps other developers understand what type of arguments are expected by the method and what type of value it returns.